src/domain/event/utils.ts

```diff
@@ -1235,6 +1248,10 @@ const getUpdateEventActionWarning = (
     return t('event.form.editButtonPanel.warningEventInPast');
   }
 
+  if (noEmailFound && action == EVENT_ACTIONS.SEND_EMAIL) {
+    return t('event.form.editButtonPanel.warningNoEmailFound');
+  }
```

Comment:
Could add a unit test to test that warning is shown if email is not found from created by attribure